core/src/main/java/com/redhat/lightblue/client/request/LightblueRequest.java

```diff
@@ -39,6 +39,7 @@ public HttpMethod getHttpMethod() {
     /**
      * Requst body as string, defaults to getBodyJson().toString()
      */
+    @Deprecated
```

Comment:
Well, it can be configured, but isn't configured the same as for responses. Details in #328